share/rocm/cmake/ROCMCreatePackage.cmake

```diff
@@ -292,20 +292,31 @@ macro(rocm_create_package)
         set(PYTHON_SITE_PACKAGES
             "/usr/lib/python3/dist-packages;/usr/lib/python2.7/dist-packages"
             CACHE STRING "The site packages used for packaging")
-        foreach(PYTHON_SITE ${PYTHON_SITE_PACKAGES})
-            file(
-                APPEND ${PROJECT_BINARY_DIR}/debian/postinst
-                "
-                mkdir -p ${PYTHON_SITE}
-                echo \"${LIB_DIR}\" > ${PYTHON_SITE}/${PARSE_NAME}.pth
-            ")
-
-            file(
-                APPEND ${PROJECT_BINARY_DIR}/debian/prerm
-                "
-                rm ${PYTHON_SITE}/${PARSE_NAME}.pth
-            ")
-        endforeach()
+        list(GET PYTHON_SITE_PACKAGES 0 PYTHON_SITE_0)
+        list(GET PYTHON_SITE_PACKAGES 1 PYTHON_SITE_1)
+        file(
+            APPEND ${PROJECT_BINARY_DIR}/debian/postinst
+            "
+            set_libdir(){
+                mkdir -p ${PYTHON_SITE_0}
+                echo \"${LIB_DIR}\" > ${PYTHON_SITE_0}/${PARSE_NAME}.pth
+
+                mkdir -p ${PYTHON_SITE_1}
+                echo \"${LIB_DIR}\" > ${PYTHON_SITE_1}/${PARSE_NAME}.pth
+            }
+            set_libdir
```

Comment:
Shouldn't ldconf be added into this function also?

Reply:
Since only a handful of packages use PYTHON path, I would suggest to keep in separate.
From what I remember ldconfig enablement is controlled by a different variable as well.

Reply:
> From what I remember ldconfig enablement is controlled by a different variable as well.

A "different" variable? There is no variable to disable the PTH files, only for ldconf. Since its a cmake variable, it requires to a different different build, right?

From what I understand is that you are just removing the `set_libdir` line from the packaging scripts for multi-rocm builds. We could do the same for ldconf, and we dont need two different builds(it also seems simpler).

I guess we can address this in a future PR though.